services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpointImpl.java

```diff
@@ -76,6 +77,9 @@ public MigrationProject getMigrationProject(Long id)
     @Override
     public MigrationProject createMigrationProject(MigrationProject migrationProject)
     {
+        if (null != getProjectIdByName(migrationProject.getTitle()))
```

Comment:
BTW it's quite natural to Java too, see  "foo".equals(value); or Boolean.TRUE.equals(bool)